docker-storage-setup.sh

```diff
@@ -16,53 +16,25 @@
 # limitations under the License.
 #++
 
-#  Purpose:  This script grows the root filesystem and sets up LVM volumes
-#            for docker metadata and data.
+#  Purpose:  This script sets up the storage for container runtimes.
 #  Author:   Andy Grimm <agrimm@redhat.com>
 
 set -e
 
 # This section reads the config file /etc/sysconfig/docker-storage-setup
-# Currently supported options:
-# DEVS: A quoted, space-separated list of devices to be used.  This currently
-#       expects the devices to be unpartitioned drives.  If "VG" is not
-#       specified, then use of the root disk's extra space is implied.
-#
-# VG:   The volume group to use for docker storage.  Defaults to the volume
-#       group where the root filesystem resides.  If VG is specified and the
-#       volume group does not exist, it will be created (which requires that
-#       "DEVS" be nonempty, since we don't currently support putting a second
-#       partition on the root disk).
-#
-# The options below should be specified as values acceptable to 'lvextend -L':
-#
-# ROOT_SIZE: The size to which the root filesystem should be grown.
-#
-# DATA_SIZE: The desired size for the docker data LV.  Defaults to using all
-#            free space in the VG after the root LV and docker metadata LV
-#            have been allocated/grown.
-#
-# Other possibilities:
-# * Support lvm raid setups for docker data?  This would not be very difficult
-#   if given multiple PVs and another variable; options could be just a simple
-#   "mirror" or "stripe", or something more detailed.
-
-# In lvm thin pool, effectively data LV is named as pool LV. lvconvert
-# takes the data lv name and uses it as pool lv name. And later even to
-# resize the data lv, one has to use pool lv name. So name data lv
-# appropriately.
-#
-# Note: lvm2 version should be same or higher than lvm2-2.02.112 for lvm
-#       thin pool functionality to work properly.
+# Read man page for a description of currently supported options:
+# 'man docker-storage-setup'
+
 POOL_LV_NAME="docker-pool"
 DOCKER_ROOT_LV_NAME="docker-root-lv"
 DOCKER_ROOT_DIR="/var/lib/docker"
+DOCKER_METADATA_DIR="/var/lib/docker"
+DOCKER_ROOT_VOLUME_SIZE=40%FREE
```

Comment:
This was being set originally by /usr/lib/docker-storage-setup/docker-storage-setup.conf, but we have removed it from their, since we don't want references to "docker" in container-storage-setup configuration.  We hard code it here so that on upgrade systems will continue to work